delegates.txt

```diff
@@ -26,6 +26,7 @@ Anne van Kesteren (AVK)
 Arnaud Le Hors (ALH)
 Ashley Hauck (AEH)
 Ashley Williams (AWS)
+Attila Bartha (AB)
```

Comment:
other than legacy examples, i believe we try to have all abbreviations be three letters.